Database/Scripts/Queries in txt/1 - authors-query.txt

```diff
@@ -0,0 +1,189 @@
+ INSERT INTO `databoom_db`.`authors` (`id`, `firstName`, `lastName`, `middleName`) VALUES (1, "S","Abid", "");
```

Comment:
Similarly here, it seems like your /Scripts for SQL queries/ will generate all these txt files. We probably only need to keep default-queries

Reply:
Overall, we can remove them, it is a personal preference to keep them, gives a better idea to those looking at our repo what the scripts give as well as to use them without the need to run the scripts themselves (for reviewal process, these texts could be ignored)

Overall, the only files in need of reviewal would be the scripts as well as the builders IMO

Reply:
Its up to you. We have documentation that describe how the scripts work. The general idea is that we should code to the repo that is mandatory, these seem to be generated easily enough.

Reply:
Agreed, thank you for pointing that out. We can remove the 'text file queries'